src/BaseTemplates/DNXEmptyWeb.vstemplate

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
   <TemplateData>
-    <Name>ASP.NET 5 Empty Web Application</Name>
-    <Description>A project for creating an empty ASP.NET 5 web application</Description>
+    <Name>ASP.NET Core 1 Empty Web Application</Name>
```

Comment:
They will show up as description for the template as shown below.

![oneaspnet](https://cloud.githubusercontent.com/assets/1132652/12656048/f5502ed8-c5b0-11e5-8041-369ee1abc693.PNG)